client/Contexts/ConfigFeatureFlag/Context.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React, { createContext, FunctionComponent, useContext } from 'react';
+import { useQuery } from '@apollo/client';
+import { GET_CONFIG } from 'Queries/Config';
+import {
+  apolloQueryResponseType,
+  ConfigFeatureFlagType,
+  defaultClientConfig,
+  defaultConfigFeatureFlagValue,
+} from './ConfigFeatureFlag.assets';
+
+const ConfigFeatureFlag = createContext(defaultConfigFeatureFlagValue);
+/** useConfigFeatureFlag - hook which returns the current value of the ConfigFeatureFlag context */
+const useConfigFeatureFlag: () => ConfigFeatureFlagType = () =>
```

Comment:
Opinions on this being exposed through the hooks folder instead?

Reply:
Fair point - think this makes sense, and also means I can get rid of the 'private' provider as I will now need to export the context as a whole (this can use the provider from that). Will update